vmdb/app/models/ems_refresh/manager.rb

```diff
@@ -0,0 +1,19 @@
+module EmsRefresh
+  module Manager
+    extend ActiveSupport::Concern
+    module ClassMethods
+      unless defined?(ems_type)
+        def ems_type
+        end
+      end
+    end
+
+    module InstanceMethods
```

Comment:
This isn't necessary anymore, and is generally avoided.  You can just use normal module mixin methods for instance methods.